soh/soh/Enhancements/randomizer/3drando/location_access/locacc_ganons_castle.cpp

```diff
@@ -123,7 +129,13 @@ void AreaTable_Init_GanonsCastle() {
   if (Dungeon::GanonsCastle.IsMQ()) {
   areaTable[GANONS_CASTLE_MQ_LOBBY] = Area("Ganon's Castle MQ Lobby", "Ganons Castle", GANONS_CASTLE, NO_DAY_NIGHT_CYCLE, {}, {}, {
                   //Exits
-                  Entrance(GANONS_CASTLE_ENTRYWAY,        {[]{return (CanUse(MASTER_SWORD) || (HasExplosives || ((Nuts || Boomerang) && (Sticks || KokiriSword))));}}),
+                  Entrance(GANONS_CASTLE_ENTRYWAY, {[]{return true;}}),
+                  Entrance(GANONS_CASTLE_MQ_MAIN,  {[]{return (CanUse(MASTER_SWORD) || CanUse(BIGGORON_SWORD || CanUse(MEGATON_HAMMER))) || ((HasExplosives || Nuts || CanUse(BOOMERANG)) && CanJumpslash);}}),
```

Comment:
looks like u missed the close bracket here. also although id be cleaning this up in my pass through later, is there any reason why Kokiri Sword isnt allowed here (so we can use CanUseSword)? to me if a section is going to be restricted to a certain age then it should be done explicitly rather than assuming for the items
```suggestion
                  Entrance(GANONS_CASTLE_MQ_MAIN,  {[]{return (CanUse(MASTER_SWORD) || CanUse(BIGGORON_SWORD) || CanUse(MEGATON_HAMMER)) || ((HasExplosives || Nuts || CanUse(BOOMERANG)) && CanJumpslash);}}),
```

Reply:
Kokiri sword cannot kill Armos alone, that is why there is the extra handling with nuts/bombs/boomerang as child.

Kokiri sword/sticks is handled via `CanJumpSlash`

Reply:
ah ok cool, just making sure, i think i noticed last time i saw this and then forgot :P